src/graph.js

```diff
@@ -57,8 +58,8 @@ export default class Graph {
     coords.length = requiredNumOfPoints;
```

Comment:
Can you change the name of this var as currently it is confusing e.g.: coords => groupedHist (or histGroups)

Reply:
Updated in commit e7f5100